<launch>
<!--
	<node	name="usb_cam"
		pkg="usb_cam"
		type="usb_cam_node"
		output="screen">
	</node>	
-->

	<arg	name	= "sub_topic"
		default	= "/stereo/right/image_raw"/>
	<arg	name	= "pub_topic"
		default	= "/stereo/right/image_raw/calib"/>
	<arg	name	= "bagfiles"
		default = "~/BAG_0901/InCheon_2016-09-01-18-25-27.bag" />
<!--
	<node	name="playbag"
		pkg="rosbag"
		type="play"
		output="screen"
		agrs="-d 3 $(arg bagfiles)" />
-->
<!--
	<node 	name	= "image_republish"
		pkg	= "image_transport"
		type	= "republish"
		output	= "screen"
		args	= "compressed in:=$(arg sub_topic) raw out:=$(arg sub_topic)"/>
-->
	<node 	name="CALIB"
		pkg="head_cam"
		type="head_cam_node"
		output="screen" >

	<param 	name="sub_topic"
		value="$(arg sub_topic)" />
	<param 	name="pub_topic"
		value="$(arg pub_topic)" />
	<param 	name="framerate"
		value="30" />
	<param  name="LUT_PARAM"
		value="0.000008205" />
	<param  name="camera_info_url"
		value="/home/a/Desktop/camera.yml" />
	<param  name="method"
		value="2" />
	<param  name="balance"
		value="0.2" />

	</node>

</launch>
